Dockerfile

```diff
@@ -6,10 +6,12 @@ LABEL image="MTProxy"
 LABEL OS="Ubuntu/latest"
 COPY container-image-root/ /
 # ARG & ENV
-ARG Secret
-ENV Secret=${Secret:-ec4dd80983dbf12d6b354cf7bcfe9a48}
-ARG Workers
-ENV Workers=${Workers:-1}
+ARG SECRET
+ENV SECRET=${SECRET:-ec4dd80983dbf12d6b354cf7bcfe9a48}
+ARG WORKERS
+ENV WORKERS=${WORKERS:-1}
+ARG MTPROTO_REPO_URL
+ENV MTPROTO_REPO_URL=${MTPROTO_REPO_URL:-https://github.com/GetPageSpeed/MTProxy}

```

Comment:
by default this project use official repository, leave him as default
https://github.com/TelegramMessenger/MTProxy